calculator.js

```diff
@@ -97,30 +99,25 @@ periodButton.addEventListener("click", function() {
 negativeValueButton.addEventListener("click", function() {
     const currentValueOfResult = result.textContent;
     if (currentValueOfResult !== "" && currentValueOfResult[0] !== "-") { result.textContent = "-" + currentValueOfResult };
-    if (currentValueOfResult !== "" && currentValueOfResult[0] === "-") { result.textContent = currentValueOfResult.slice(1) };
-
+    if (currentValueOfResult !== "" && currentValueOfResult[0] === "-") { result.textContent = currentValueOfResult.slice(0) };
 });
 
 addingButton.addEventListener("click", function() {
```

Comment:
You're not using `_firstOperand ` in the operator functions 
you need to update the value of `_firstOperand ` with the operator type

Reply:
yes i am 
i will think how